src/sentry/projectoptions/defaults.py

```diff
@@ -44,6 +44,12 @@
 # is auto upgrading enabled?
 register(key="sentry:grouping_auto_update", default=True)
 
+# Has this project had its issues backfilled into the Seer database, and if so, when did the
+# backfill complete? (This is a temporary way to flag projects as we roll out Seer grouping, because
+# it can be flipped on in the backfill script, unlike inclusion in a getsentry feature handler.)
+register(key="sentry:similarity_backfill_completed", default=None)
```

Comment:
wouldn't that also mean that it's visible and modificable by users? Or does `sentry:` prefix prevent that?

Reply:
I don't think any option just automatically shows up in project settings, does it? Pretty sure we have to build UI for it, don't we?

I will spend a few minutes poking around just to be sure before I merge this, but I'm 99% certain everything is internal until we explicitly do work to make it external.

Reply:
Yeah, we have to specifically list out the fields in our forms: https://github.com/getsentry/sentry/blob/9db72d0e3ad8e0a600b08c6dc6920955ccf825c9/static/app/data/forms/projectIssueGrouping.tsx#L50